website/docs/addon_slack_admin.md

```diff
@@ -146,10 +146,10 @@ to message content instead of using `Upload review`. This link might help users
 :::
 - **Message:**
 Message content can use template keys (see [Available template keys](admin_settings_project_anatomy.md#available-template-keys)).
-Few keys also have Capitalized and UPPERCASE format. Values will be modified accordingly. e.g. `{Folder[name]}` ➜ "Gun", `{PRODUCT}` ➜ "RENDER".
+Few keys also have Capitalized and UPPERCASE format. Values will be modified accordingly. e.g. `{Folder[name]}` ➜ "Gun", `{PRODUCT[name]}` ➜ "RENDER".
 :::tip Message Example
   ```
-  {Product} was published for {FOLDER[NAME]} in {task[name]} task.
+  {Product[name]} was published for {FOLDER[NAME]} in {task[name]} task.
```

Comment:
ehm, so whatever inside the square brackets should be in small case? 
```suggestion
  {Product[name]} was published for {FOLDER[name]} in {task[name]} task.
```

Reply:
Actually no 
![image](https://github.com/user-attachments/assets/2827100d-768e-4504-b466-2db16426a528)
I'll fix occurrence before this one.